docs/usage.md

```diff
@@ -52,6 +52,14 @@ Some options are **global only**, and go directly under the handler's name.
 
     More details at [Finding modules](#finding-modules).
 
+- `preload_external_modules`: this option is used to load external modules that are
+   not specified directly, in order to resolve aliases. For example, if you define
+   a class `C` in package `A` and import it in package `B`, you can explicitly export
+   it by defining `__all__=['C']` in package B, and then the documentation for
+   class `C` will appear under package `B`, even if `A` documentation is not part of
+   the classes given to mkdocstrings to document. (i.e. even if `::: A` does not appear in
+   the documentation). The modules should be listed as an array of strings, e.g. `['A', 'B']`.
+
```

Comment:
The whole paragraph is hard to read and understand. I suggest something different:

```suggestion
- `preload_modules`: this option is used to pre-load modules that are
   not specified directly in autodoc instructions (`::: identifier`).
   It is useful when you want to render documentation for a particular member of an object,
   and this member actually comes from another package than its parent.
   The modules must be listed as an array of strings.

```

Reply:
Oh by the way since it's a global *and* local option, could you move it in the relevant section?

Reply:
Done